menpo/feature/features.py

```diff
@@ -768,6 +768,58 @@ def lbp(pixels, radius=None, samples=None, mapping_type='riu2',
 
 
 @ndfeature
+def normalize(pixels, scale_func=None, mode='all'):
+    r"""
+    Normalize the pixel values via mean centring and an optional scaling. By
+    default the scaling will be ``1.0``. The ``mode`` parameter selects
+    whether the normalisation is computed across all pixels in the image or
+    per-channel.
+
+    Parameters
+    ----------
+    pixels : :map:`Image` or subclass or ``(C, X, Y, ..., Z)`` `ndarray`
+        Either the image object itself or an array with the pixels. The first
+        dimension is interpreted as channels. This means an N-dimensional image
+        is represented by an N+1 dimensional array.
+    scale_func : `callable`, optional
+        Compute the scaling factor. Expects a single parameter and any number
```

Comment:
No, the single arg is the pixel array right? Although I'm a little hazy on why this needs to be a callable (over simply a keyword for the scale factor directly)..is there a motivating example?